module-2/lab-hypothesis-testing/your-code/challenge-2.ipynb

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
- Siempre es recomendable printear todos los resultados para obtener además del p valor el estadístico de contraste. 
```Python
anova=stats.f_oneway(*pokemon_groups)
print(anova)
```
- "First let's obtain the unique values of the pokemon types. These values should be extracted from Type 1 and Type 2 aggregated. Assign the unique values to a variable called unique_types.
Hint: the correct number of unique types is 19 including NaN. You can disregard NaN in next step." Otra forma de hacerlo muy simplota sería:
```Python
types = pokemon['Type 1'].append(pokemon['Type 2']).unique()
unique_types = types.tolist()
len(unique_types)
```
- Mejor que "Hypothesis Zero (H0) HYPER REFUTED", rechazamos la hipótesis nula a diferentes niveles de significación, 0,01, 0.05,0.1,....